tas2781-fix.sh

```diff
@@ -299,7 +309,8 @@ run_fix_service() {
   local props_changed='select(.info["change-mask"]|index("props"))'
   local sink='select(.info.props["media.class"]=="Audio/Sink")'
   local snd_hda_intel='select(.info.props["alsa.driver_name"]=="snd_hda_intel")'
-  local alsa_components='select(.info.props["alsa.components"]|test("17aa3886"))'
+  local alsa_component_id=$(pw-dump | jq -r 'first(.[] | select(.info?.props?["alsa.card_name"] == "HDA Intel PCH")) | .info.props["alsa.components"] | split(",")[1]')
```

Comment:
There doesn't need to be an extra step to parse out  `alsa.component_id` here. The better solution would be to just replace the check for `alsa.component_id`  with a check for `alsa.card_name` in the main `pw-dump` loop

Reply:
I just tried that and it seems to work, that's much better

Reply:
Great, so once you make that change and the one in the comment thread above, I'm happy to approve and merge. Thanks for doing this!